### PR TITLE
Wrote tests for Customer Entity

### PR DESCRIPTION
**Pull request by Andreas-Kreouzos**
Added unit and integration tests across the Customer entity

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ target/
 !**/src/main/**/target/
 !**/src/test/**/target/
 application.yml
+docker-compose.yml
```
Comment (amigoscode) [inline .gitignore:5]: remove this

**Review by ghost — COMMENTED** [review]
this pr contains more than tests. are you sure is this the right changes?

File: .gitignore (hunk 1 above)
Comment (Andreas-Kreouzos) [reply .gitignore:5]: Done

**Comment by Andreas-Kreouzos** [comment]
My mistake, I had to make some changes in the project as well as the tests that took place. I made another push today with all the final changes as well as tests. Added some more integration tests that were missing. Sorry for the confusion.

**Review by MoeAlshidi — DISMISSED** [review]

**Review by MoeAlshidi — APPROVED** [review]

**Pull request merged**